src/generator.py

```diff
@@ -101,44 +102,26 @@ def get_type_string(self, t, pointer=False, name=''):
     def is_vector(self, t):
         return ('vector' in t)
 
-    def capitalize(self, input_string):
-        capitalized_words = [ x[0].capitalize() + x[1:] for x in input_string.split('_')]
-        return ''.join(capitalized_words)
+    def generate(self, definition, template, globuls=None):
```

Comment:
What is a "globul"? should it be "glob"?

Reply:
`global` is a python keyword